docker/witnet-rust/runner.sh

```diff
@@ -19,10 +19,48 @@ URL="https://github.com/witnet/witnet-rust/releases/download/$VERSION/witnet-$VE
 
 FILENAME="$VERSION.tar.gz"
 FOLDERNAME="."
+CONFIG_FILE_FROM_CMD=`echo "$@" | sed -E 's/(.*-c\s*)?(.*\.toml)?.*/\2/'`
+CONFIG_FILE=${CONFIG_FILE_FROM_CMD:-witnet.toml}
```

Comment:
I have updated my commit so that the path for the config file is not hard-coded: it reads it now from the command line, i.e. it parses `-c /whatever/path/is/the/config.toml` :wink: